### ISSUE TITLE
MountVolume.SetUp failed for volume "certs" : secret "ess-server-certs" not found

### ISSUE THREAD

**Issue by Nold360** [crossplane-contrib/ess-plugin-vault#7]
### What happened?

I applied the chart es recommended but the pod is failing because of a missing secret: 

```
MountVolume.SetUp failed for volume "certs" : secret "ess-server-certs" not found
```
### How can we reproduce it?

```shell
cat > values.yaml <<EOF
podAnnotations:
  vault.hashicorp.com/agent-inject: "true"
  vault.hashicorp.com/agent-inject-token: "true"
  vault.hashicorp.com/role: crossplane
  vault.hashicorp.com/agent-run-as-user: "65532"
EOF

helm upgrade --install ess-plugin-vault oci://xpkg.upbound.io/crossplane-contrib/ess-plugin-vault --namespace crossplane-system -f values.yaml
```

### What environment did it happen in?
Crossplane version: 1.13.1

**Comment by ezgidemirel** [comment]
Hi @Nold360, Crossplane is responsible for creating that secret and you need to enable the beta feature with `--enable-external-secret-store` flag before installing the plugin.

**Comment by ehni** [comment]
Uninstalling and reinstalling the plugin didn't solve the issue for me unfortunately.

**Comment by amila77kashyapa** [comment]
Hello All, 

Has anyone found a solution for this? I'm experiencing it with version 1.15.0. 'ess-server-certs' secret was not created even though I enabled the '--enable-external-secret-store' feature flag.

**Comment by amila77kashyapa** [comment]
@ezgidemirel not sure is this updated "oci://xpkg.upbound.io/crossplane-contrib/ess-plugin-vault" after "https://github.com/crossplane/crossplane/pull/3884#event-9985335927" this change. ess-plugin-vault chart still looking for this "ess-server-certs". EX: After changing "ess-plugin-vault" certs volume secret to "crossplane-tls-server" ess-plugin-vault deployed successfully. Is there any define solution for this. Thank you.

**Comment by ezgidemirel** [comment]
Hello @amila77kashyapa, `crossplane-tls-server` certificate is used for crossplane itself. The ESS plugin should mount the `ess-server-certs` secret. 

I have run the following command to enable external secret stores functionality on my Crossplane and verified that the `ess-server-certs` secret is created on the `crossplane-system` namespace. If you deploy your plugin to crossplane-system, it should work. 

```
helm upgrade crossplane --namespace crossplane-system crossplane-stable/crossplane --set args={--enable-external-secret-stores}
```

If the secret is not created on your setup, could you please share the steps you applied for me to reproduce? Thanks.

**Comment by amila77kashyapa** [comment]
@ezgidemirel  Thank you, I found the issue. I used the command below: "helm install crossplane --namespace crossplane-system crossplane-stable/crossplane --set args='{"--enable-external-secret-stores"}' -f aws-values.yaml"  which refers to the Crossplane installation document with the feature flag. It looks like the double quotes caused the issue. The secret is being created without any issues using the command you provided.
![image](https://github.com/crossplane-contrib/ess-plugin-vault/assets/57173475/95e51bd1-881e-4bb8-aef7-a8f5cf3cd00e)
